Jenkinsfile

```diff
@@ -228,6 +233,18 @@ def test(task) {
 
 }
 
+def runGenerateDocs() {
+    withEnv(['PATH+USER_BIN=/usr/local/bin']) {
+        node(osx_kotlin) {
+            sh """
+                cd packages
+                ./gradlew --no-daemon :library:dokkaJar --info --stacktrace
+            """
+            archiveArtifacts artifacts: 'packages/library/build/libs/library-*-dokka.jar'
```

Comment:
I don't think we normally archive the docs, but I guess it doesn't hurt either 👍

Reply:
It was just to have the generated artifact somewhere for inspection. I removed it in favor of adding the artifact to the publication.